.eslintrc.json

```diff
@@ -51,20 +51,16 @@
       "error",
       "global"
     ],
-    "@typescript-eslint/ban-types": [
-      "error",
-      {
-        "types": {
-          "Function": false
-        }
-      }
-    ],
+    "@typescript-eslint/no-unsafe-function-type": "off",
```

Comment:
Superceeded by `no-unsafe-function`, which is enabled by default from the typescript-eslint recommended rules package.